docs/widgets/masked_input.md

```diff
@@ -0,0 +1,84 @@
+# MaskedInput
+
+!!! tip "Added in version 0.74.0"
```

Comment:
Just to note that this version will need updating before release

Reply:
Cheers @TomJGooding - I've updated that now.